README.md

```diff
@@ -1,22 +1,15 @@
 Woketo
 ======
 
-**A websocket library designed to be released.**
+A PHP WebSocket library. With following features:
 
-> This means there is no version  that is going to be release until [v1 milestone](https://github.com/Nekland/Woketo/milestone/1) is complete.
+* Autobahn test suite passed (WebSocket test suite reference)
+* Binary/text messages supported
+* Built on top of reactphp (async socket communication)
+* *Not dependent of any other big framework/library which mean you can use with guzzle (any version) or Symfony (any version)*
```

Comment:
* *Not dependent of any other big framework/library which means you can use *it* with guzzle (any version) or Symfony (any version)*